docs(proceso): adición de bitácoras de weeklies y documento DOD

manriqueespinosaleonardo opened the pull request:
**Trabajo completado:**
- Se registraron las bitácoras de los weeklies
- Se redactó el documento de Definition of Done

**Archivos principales:**
- `Proyecto Final/proceso/bitacoras/weekly-semanal-del-10-al-16-de-noviembre.md`
- `Proyecto Final/proceso/bitacoras/weekly-semanal-del-17-al-23-de-noviembre.md`
- `Proyecto Final/proceso/bitacoras/weekly-semanal-del-24-al-28-de-noviembre.md`
- `Proyecto Final/docs/requisitos/dod.md`

**Checklist:**
- [x] Revisé ortografía y gramática
- [x] Los archivos siguen la [convención de nombres](../CONTRIBUTING.md#nomenclatura)
- [x] Verifiqué que todo mi trabajo funcione
- [x] No incluí archivos temporales o innecesarios

manriqueespinosaleonardo committed:
docs(requisitos/proceso): actualización de documentación

**Trabajo completado:**
- Adición de las bitácoras de las reuniones 3, 4 y 5
- Adición de documentación de artefactos, herramientas y procesos

**Archivos principales:**
- `Proyecto Final/docs/proceso/bitacoras/reunion3.md`
- `Proyecto Final/docs/proceso/bitacoras/reunion4.md`
- `Proyecto Final/docs/proceso/bitacoras/reunion5.md`
- `Proyecto Final/docs/proceso/bitacoras/reunion3.md`
- `Proyecto Final/docs/proceso/proceso-del-equipo.md`
- `Proyecto Final/docs/requisitos/artefactos/artefactos-del-equipo.md`
- `Proyecto Final/docs/requisitos/herramientas/herramientas-del-equipo.md`

manriqueespinosaleonardo committed:
Pruebas de Uso y actualización de los casos de uso

Se realizó el documento de las pruebas de uso que se implementaran esta semana

scottgeliza committed:
Merge branch 'Entrega-3-FIS' into Leo-Manrique

manriqueespinosaleonardo committed:
Bitácoras y DoD

Agregue las bitácoras de los weeklys y el DoD

manriqueespinosaleonardo committed:
Merge branch 'Leo-Manrique' of https://github.com/melismau/FIS-Cambranes-EQ1 into Leo-Manrique

scottgeliza commented on Proyecto Final/docs/requisitos/dod.md:
should be named just dod.md as per your pull request description. mayapprendo is kind of redundant

scottgeliza reviewed (changes requested):
these are outdated and reflect weeklies from previous months, not this one.
also the format doesn't match up to the previous ones, they were called reunion1, reunion2, reunion3, so on so forth

manriqueespinosaleonardo committed:
Corrección de archivos

Corrección de los archivos de las bitácoras y cambio de nombre en los archivos de weeklys y dod

scottgeliza reviewed (approved):
looks good this time
